cron/class-wcal-cron.php

```diff
@@ -270,7 +275,7 @@ public function wcal_send_email_notification() {
 														$cart_page_link = $cart_page_id ? get_permalink( $cart_page_id ) : '';
 													}
 
-													$encoding_cart   = $email_sent_id . '&url=' . $cart_page_link;
+													$encoding_cart   = $email_sent_id . '&url=' . $cart_page_link . $utm;
```

Comment:
Wouldn't this result in a query string like: `&url=<cartpagelink>?<utm parameter string>`. If so, it would be incorrect to have the `?` appear after the `&` in the query string.

Reply:
No, it would not be incorrect. When the link is accessed the content that starts from after the = sign of url= is taken as the whole URL. So the part before that is removed and the redirect is performed only to the `<cartpagelink>?<utm` parameter string>`